src/MLOps.NET/Storage/Database/MLOpsDbContext.cs

```diff
@@ -79,5 +79,10 @@ public virtual void EnsureCreated()
 
         ///<inheritdoc cref="IMLOpsDbContext"/>
         public DbSet<Deployment> Deployments { get; set; }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public DbSet<ModelSchema> Schemas { get; set; }
```

Comment:
I think we can remove this DbSet and instead add the ModelSchema a property to a Run instead (e.g. `public List<ModelSchema> ModelSchemas {get; set;}